ocs_ci/framework/pytest_customization/ocscilib.py

```diff
@@ -464,6 +481,15 @@ def gather_version_info_for_report(config):
         mods = {}
         mods = get_version_info(namespace=ocsci_config.ENV_DATA["cluster_namespace"])
         skip_list = ["ocs-operator"]
+        if ocsci_config.MULTICLUSTER["primary_cluster"]:
+            config._metadata["OADP Operator"] = get_oadp_version()
+            config._metadata[
+                "OCP DR Cluster Operator"
+            ] = get_ocp_dr_cluster_operator_version()
+            config._metadata["GitOps Operator"] = get_ocp_gitops_operator_version()
+            config._metadata["VolSync Operator "] = get_volsync_operator_version()
```

Comment:
The same question about diff between RDR/MDR like above.

Reply:
@sidhant-agrawal